includes/class-boldgrid-editor-assets.php

```diff
@@ -163,6 +163,31 @@ public static function is_webpack() {
 		return ( defined( 'BGEDITOR_SCRIPT_DEBUG' ) && BGEDITOR_SCRIPT_DEBUG );
 	}
 
+	/**
+	 * Maybe Enqueue Animate.css
+	 *
+	 * Checks if post uses the animate.css or not.
+	 *
+	 * @since SINCEVERSION
+	 *
+	 * @return bool True if animate.css is to be enqueued.
+	 */
+	public function maybe_enqueue_animate() {
+		global $post;
+
+		if ( ! $post ) {
+			return true;
+		}
+
+		$post_content = apply_filters( 'the_content', $post->post_content );
```

Comment:
Why are we running this filter? Are the animations added via a filter?

Reply:
apply_filters( 'the_content' ) is the best way to ensure that the content we are testing is the most accurate, as it will have any and all content filters applied. using the 'the_content' filter is preferred over 'get_the_content()'

[For example, using 'get_the_content()' will not expand shortcodes, among other things](https://developer.wordpress.org/reference/functions/get_the_content/#:~:text=An%20important%20difference%20from%20the_content,expand%20shortcodes%2C%20among%20other%20things.)